.github/workflows/update-plugin.yml

```diff
@@ -0,0 +1,70 @@
+name: Create Plugin Revision
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'Plugin Repository Name'     
+        required: true
+      target-branch:
+        description: 'Target Branch'
+        default: master
+        required: false
+jobs:
+  create_revision_pr:
+    runs-on: ubuntu-latest
+    name: Create Plugin Revision
+
+    steps:
+    # Step 1: run a standard checkout action, provided by github
+    - name: Checkout HEAD
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.inputs.target-branch }}
+    
+    # Step 2: Echo inputs
+    - name: Read release information
+      id: plugin_info
+      env:
+        PLUGIN_NAME: ${{ github.event.inputs.repository }}
+      run: |
+        RELEASE=$(curl -s https://api.github.com/repos/${{ github.repository_owner }}/${PLUGIN_NAME}/releases/latest | grep tag_name | cut -d '"' -f 4) 
```

Comment:
This assumes plugin code is in Github, uses tagging and has the same version numbering scheme.

Reply:
Current iteration assumes that the plugin is indeed the result of a sem-ver release on a GitHub repo. The `archive` endpoint is more consistent than the `releases/download` - but the GitHub Action could be extended to look for the alternative (just add a flag with `archive` being the default.)

Reply:
Fix d6e7c32 - you can now optionally add `url` and `vers` directly into the action and the PR will calculate the checksum. If not supplied the default is to curl for the latest release.